test/officerManager.js

```diff
@@ -219,8 +216,7 @@ describe('OfficerManager', () => {
           expect(response).to.have.length(1)
           expect(response[0].email).to.be.eql('test@test.com')
           expect(response[0].name).to.be.eql('pinkUnicorn')
-          expect(response[0].level).to.be.eql(2)
-          expect(response[0].team).to.be.eql('dev')
+          expect(response[0].role).to.be.eql('dev')
```

Comment:
```suggestion
          expect(response[0].role).to.be.eql(form.role)
```

Reply:
do this for all the other test cases too. we shouldnt be hardcoding the test cases